docs/index-shared.asciidoc

```diff
@@ -0,0 +1,67 @@
+
+:release-state:  unreleased
+:version:        5.5.0
+:major-version:  5.x
+:branch:         5.x
+
+:docker-image:   docker.elastic.co/kibana/kibana:{version}
+:es-ref:         https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/
+:xpack-ref:      https://www.elastic.co/guide/en/x-pack/current/
+:repo:           https://github.com/elastic/kibana/
+:issue:          {repo}issues/
+:pull:           {repo}pull/
+:commit:         {repo}commit/
+:blob:           {repo}blob/{branch}/
+:security:       https://www.elastic.co/community/security/
+:xpack:             X-Pack
+:security:          X-Pack security
+:monitoring:        X-Pack monitoring
+:watcher:           Watcher
+:reporting:         X-Pack reporting
+:graph:             X-Pack graph
+:searchprofiler:    X-Pack search profiler
+:xpackml:           X-Pack machine learning
+:es:                Elasticsearch
+:kib:               Kibana
+
+ifeval::["{include-xpack}"=="true"]
+:xkb-repo-dir: {docdir}/../../x-pack-kibana/docs/en
+endif::[]
+
+include::introduction.asciidoc[]
+
+include::setup.asciidoc[]
+
+include::migration.asciidoc[]
+
+include::getting-started.asciidoc[]
+
+include::discover.asciidoc[]
+
+include::visualize.asciidoc[]
+
+include::dashboard.asciidoc[]
+
+include::timelion.asciidoc[]
+
+//include::{xes-repo-dir}/ml/index.asciidoc[]
```

Comment:
Let's exclude the commented out sections that don't exist yet unless there is a specific reason to keep them in.  We avoid merging commented out docs or code because it's not clear to future committers why those things are commented out.